automatic/intunewinapputil/intunewinapputil.nuspec

```
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>intunewinapputil</id>
+    <version>1.8.4</version>
+    <packageSourceUrl>https://github.com/auto</packageSourceUrl>
+    <owners>chocolatey-community,corbob</owners>
+    <title>intunewinapputil</title>
+    <authors>Microsoft</authors>
+    <projectUrl>https://github.com/microsoft/Microsoft-Win32-Content-Prep-Tool</projectUrl>
+    <iconUrl>https://raw.githubusercontent.com/auto</iconUrl>
+    <licenseUrl>https://github.com/microsoft/Microsoft-Win32-Content-Prep-Tool/raw/master/Microsoft%20License%20Terms%20For%20Win32%20Content%20Prep%20Tool.pdf</licenseUrl>
```

```suggestion
    <licenseUrl>https://github.com/microsoft/Microsoft-Win32-Content-Prep-Tool/blob/master/Microsoft%20License%20Terms%20For%20Win32%20Content%20Prep%20Tool.pdf</licenseUrl>
```

Try avoiding direct URLs to the file, especially when GitHub can display these without having to download the file.